main/acle.md

```diff
@@ -2370,7 +2371,7 @@ compiler and it is enabled.
 
 ### Name mangling
 
-The `"default"` version is not mangled top of the languge specific name
+The `"default"` version is not mangled top of the language specific name
 mangling.
```

Comment:
Are there missing words between mangled and top?
should it read
"The '"default"' version is not mangled at the top of the language specific name mangling